utils.py

```diff
@@ -27,97 +28,73 @@ def hit_detection_and_move_bullets(
         if bt.move_bullet(dt):
             bullets.remove(bt)
             continue
-        for orb in balloons:
+        for orb in targets:
             if bt.rect.colliderect(orb.rect):
                 bullets.remove(bt)
-                balloons.remove(orb)
+                targets.remove(orb)
                 
                 
 def hit_collision_player(
-        balloons: list[balloon.Balloon], 
-        player : aircraft.Aircraft
+        targets: list[balloon.Balloon], 
```

Comment:
is dit ook aangepast op de plekken waar je de fucntie aanroept? Dat je de parameters hebt geswitcht?

Reply:
![afbeelding](https://github.com/Finnyy012/TUPOLEVGAMING/assets/94832237/99210a7e-eb2d-4c3a-931a-03fa22821219)